### DISCUSSION TITLE
GitHub Actions has encountered an internal error when running your job.

### DISCUSSION THREAD

**Discussion by deershark** [community/community#111945] [Actions]
### Select Topic Area

Bug

### Body

I have migrated a repository tranform from a personal repository to an organization, but now when the actions are run it tells me: "GitHub Actions has encountered an internal error when running your job.", and it has no log. When it in my personal repository, it works normal. I try very times, but it did not work.
<img width="807" alt="image" src="https://github.com/community/community/assets/29668663/77e99224-c2e5-4fe1-90d6-861ecf89bb80">

**Comment by RadxaYuntian** [comment]
I also moved a repository between 2 organizations, and I'm now seeing the exact same error.
